### PR TITLE
Tabs component

### PR DESCRIPTION
**Pull request by crperezchica**
Adds in Tabs.js
Tabs is a display component that iterates through markdown to create a list of markdown files.

### CODE REVIEW

File: src/components/markdown/Tabs.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+function Tabs({ markdownFiles }) {
+  const markdownList = markdownFiles.map((markdown, i) => {
+    return <li key={i}>{markdown}</li>;
+  });
+
+  return (
+    <>
+    <ul>
+      <li>{markdownList}</li>
```
Comment (MariahAdams) [inline src/components/markdown/Tabs.js]:
```suggestion
      {markdownList}
```
Lines 5-7 take care of creating the li element.

File: src/containers/markdown/MarkdownContainer.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import Preview from '../../components/markdown/Preview';
 import styles from './MarkdownContainer.css';
 import { updateMarkdown } from '../../actions/markdownActions';
 
+
```
Comment (mehtaphysical) [inline src/containers/markdown/MarkdownContainer.js:9]: why add this?

File: src/components/markdown/Tabs.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Tabs({ markdownFiles }) {
```
Comment (MariahAdams) [inline src/components/markdown/Tabs.js]: Do you need to be passing down the entire markdown file?

File: src/components/markdown/Tabs.js (hunk 1 above)
Comment (crperezchica) [reply src/components/markdown/Tabs.js]: implemented comment

File: src/components/markdown/Tabs.js (hunk 3 above)
Comment (crperezchica) [reply src/components/markdown/Tabs.js]: implemented change

File: src/containers/markdown/MarkdownContainer.js (hunk 2 above)
Comment (crperezchica) [reply src/containers/markdown/MarkdownContainer.js:9]: We are unsure as to what you mean. In our file we are using both styles and updateMarkdown

**Review by MariahAdams — APPROVED** [review]
Good work!

**Pull request merged**